db/migrate/20160126151325_add_disk_and_memory_minimum_to_hardware.rb

```diff
@@ -0,0 +1,6 @@
+class AddDiskAndMemoryMinimumToHardware < ActiveRecord::Migration
+  def change
+    add_column :hardwares, :disk_size_minimum, :integer, :limit => 8
```

Comment:
@Ladas I decided to stay with bytes for consistency with the other disk size columns.  The raw Openstack image objects `min_disk` is in gigabytes, so I multiply by `1.gigabyte` to get bytes.